github-push/handler_test.go

```diff
@@ -55,21 +55,3 @@ func Test_Handle_IssueComment(t *testing.T) {
 		t.Fail()
 	}
 }
-func Test_Handle_ValidateCustomers_Matched(t *testing.T) {
```

Comment:
Removed this test because It's intent is to test customers from URL but it actually gives error connected with secret, which I think should not be the case.

Reply:
Can you restore it please?

Reply:
Ok no problems